modules/linear_boltzmann_solvers/lbs_solver/volumetric_source/volumetric_source.cc

```diff
@@ -0,0 +1,125 @@
+// SPDX-FileCopyrightText: 2024 The OpenSn Authors <https://open-sn.github.io/opensn/>
+// SPDX-License-Identifier: MIT
+
+#include "modules/linear_boltzmann_solvers/lbs_solver/volumetric_source/volumetric_source.h"
+#include "modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.h"
+#include "framework/mesh/cell/cell.h"
+#include "framework/mesh/mesh_continuum/mesh_continuum.h"
+#include "framework/mesh/logical_volume/logical_volume.h"
+#include "framework/math/functions/spatial_material_function.h"
+#include "framework/runtime.h"
+#include "framework/object_factory.h"
+
+namespace opensn::lbs
+{
+
+OpenSnRegisterObjectInNamespace(lbs, VolumetricSource);
+
+InputParameters
+VolumetricSource::GetInputParameters()
+{
+  InputParameters params = Object::GetInputParameters();
+
+  params.SetGeneralDescription("General implementation of an arbitrary multi-group "
+                               "distributed source. A distributed source is given by a "
+                               "logical volume and a Lua function handle. Currently, only "
+                               "isotropic distributed sources are allowed.");
+  params.SetClassName("Distributed Source");
```

Comment:
distributed was forgotten in the replace all

Reply:
Good catch!